regression/cbmc-java/ClassCastException2/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
```

Comment:
Yes, they somehow were, thanks for noticing that. I've now replaced CCE2 by the one you suggest below (with C<:B<:A), if I understood it correctly.